src/popup/router/components/TokenAmount.vue

```diff
@@ -12,12 +12,12 @@ import { mapState } from 'vuex';
 
 export default {
   props: {
-    amount: { type: Number, required: true },
+    amount: { type: [Number, String], required: true },
```

Comment:
I would prefer a more strict validation. Add the explicit conversion to number where it is needed instead.